packages/token-lists/src/top-100.ts

```diff
@@ -3,6 +3,11 @@ import path from "path";
 import { request, gql } from "graphql-request";
 import { getAddress } from "@ethersproject/address";
 
+const pathToImages = process.env.CI
+  ? path.join(process.env.GITHUB_WORKSPACE, "packages", "token-lists", "lists", "images")
+  : path.join(path.resolve(), "lists", "images");
+const logoFiles = fs.readdirSync(pathToImages);
```

Comment:
Logic taken from tests: https://github.com/pancakeswap/pancake-toolkit/blob/master/packages/token-lists/test/default.test.ts#L92

Reply:
do we ever run top-100.ts in CI? 🤔 
The check for env.CI seems unnecessary if we do it only locally

Reply:
Planning to get a GitHub Actions up and running updating this list on a weekly basis